cpp/doAll_fitDimuonMass.C

```diff
@@ -242,7 +286,8 @@
        TString sample = sigsamples[isample];
        cout<<"Sample: "<< sample << endl;
        if ( useSignalMC ) {
-         TString inFile = Form("%s/histograms_%s_%s_%s_0.root",inDir.Data(),sample.Data(),era.Data(),year.Data());
+         //TString inFile = Form("%s/histograms_%s_%s_%s_0.root",inDir.Data(),sample.Data(),era.Data(),year.Data());
+	 TString inFile = Form("%s/histograms_%s_%s_0.root",inDir.Data(),sample.Data(),year.Data()); //changed by Javier, sample names had 2022 three times
```

Comment:
Sample names have 2022 three times instead of two because you added an extra 2022 here:
https://github.com/cmstas/run3_scouting/blob/2ca97af766400ce1b61013f6b4667a160dc3aa25/cpp/doAll_fitDimuonMass.C#L200
I remove that and revert this change